packages/browser-crawler/src/internals/browser-launcher.ts

```diff
@@ -58,6 +58,10 @@ export interface BrowserLaunchContext<TOptions, Launcher> extends BrowserPluginO
      */
     userAgent?: string;
 
+    /**
+     * The type of browser to be launched.
+     * By default, `chromium` is used. Other browsers like `webkit` or `firefox` can be used.
+     */
```

Comment:
Thanks!!!

I would also add a code example of how to import it. You can use typescript in these comments. The code can be partially copied from here
https://crawlee.dev/docs/examples/playwright-crawler-firefox

Reply:
Added [here](https://github.com/apify/crawlee/pull/1594/commits/7b2ede4bf9587331ca6578a40db82ac2ee3f9f20).